client_cli/src/main.rs

```diff
@@ -115,41 +125,58 @@ fn main() -> Result<()> {
         config: Configuration(config),
         subcommand,
     } = Args::from_args();
-
-    println!("Value for config: {:?}", &config);
-    subcommand
-        .run(&config)
-        .wrap_err("Failed to run subcommand")?;
+    println!("Iroha Client CLI: build v0.0.1 [release]");
+    println!(
+        "Logged in as {}@{}",
+        config.account_id.name, config.account_id.domain_name
+    );
+    #[cfg(debug_assertions)]
+    eprintln!(
+        "{}",
+        &serde_json::to_string(&config).wrap_err("Configuration was already verified.")?
+    );
+    #[cfg(not(debug_assertions))]
+    eprintln!("This is a release build, debug information omitted from messages");
+    subcommand.run(&config)?;
     Ok(())
 }
 
 /// # Errors
 /// Fails if submitting over network fails
+#[allow(clippy::shadow_unrelated)]
 pub fn submit(
     instruction: impl Into<Instruction>,
     cfg: &ClientConfiguration,
     metadata: UnlimitedMetadata,
 ) -> Result<()> {
     let instruction = instruction.into();
     let mut iroha_client = Client::new(cfg);
+    #[cfg(debug_assertions)]
+    let err_msg = format!(
+        "Failed to build transaction from instruction {:?}",
+        instruction
+    );
+    #[cfg(not(debug_assertions))]
+    let err_msg = "Failed to build transaction.";
     let tx = iroha_client
         .build_transaction(vec![instruction], metadata)
-        .wrap_err("Failed to build transaction.")?;
+        .wrap_err(err_msg)?;
     let tx = match iroha_client.get_original_transaction(
         &tx,
         RETRY_COUNT_MST,
         RETRY_IN_MST,
     ) {
         Ok(Some(original_transaction)) if Confirm::new()
-            .with_prompt("There is a similar transaction from your account waiting for more signatures. Do you want to sign it instead of submitting a new transaction?")
+            .with_prompt("There is a similar transaction from your account waiting for more signatures. This could be because it wasn't signed with the right key. Do you want to sign this transaction (yes) instead of submitting a new transaction (no)?")
```

Comment:
`This could be because it wasn't signed with the right key` - Or it is an MST transaction.